presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -175,6 +175,17 @@ Statistical Functions
     Returns the upper bound of the Wilson score interval of a Bernoulli trial process
     at a confidence specified by the z-score ``z``.
 
+.. function:: KLEINBERG_BURST_DETECTION(time[, s, gamma]) -> array<int>
+
+    Given an array of sorted timestamps, returns an array with the same length
+    as the input where a 0 value indicates no-burst, and any other value
+    indicates that the timestamp at this index corresponds to a burst whose
+    center is located in the index pointed by the value. The default value for
+    ``s`` is ``2.0`` and for ``gamma`` is ``1.0``::
```

Comment:
Let's end the last sentence in a period, so that it doesn't look like the example applies to the last sentence only. Write the last line like this:
```
``s`` is ``2.0`` and for ``gamma`` is ``1.0``. ::